mne/viz/utils.py

```diff
@@ -1685,10 +1677,17 @@ def _compute_scalings(scalings, inst):
         data = inst._data.swapaxes(0, 1).reshape([len(inst.ch_names), -1])
     # Iterate through ch types and update scaling if ' auto'
     for key, value in scalings.items():
-        if value != 'auto':
-            continue
         if key not in ch_types.keys():
-            raise ValueError("Sensor {} doesn't exist in data".format(key))
```

Comment:
So, there will be no warning if a channel type is mistakenly specified?

Reply:
Correct. Not sure it's worth the effort, since you could for example want to use a single scaling dict and then pick channels and plot (which would make it so your dict had unnecessary entries)